CHANGELOG.md

```diff
@@ -8,8 +8,8 @@
 - Implement the MSMR model ([#3116](https://github.com/pybamm-team/PyBaMM/pull/3116))
 
 ## Bug fixes
-
 - Fixed bug causing incorrect activation energies using `create_from_bpx()` ([#3242](https://github.com/pybamm-team/PyBaMM/pull/3242))
```

Comment:
Can you move this line down so the PRs are sorted in decreasing order? Happy to merge after that

Reply:
Done!